src/bijectors/corr.jl

```diff
@@ -348,13 +348,12 @@ function _inv_link_chol_lkj(Y::AbstractMatrix)
     T = float(eltype(W))
     logJ = zero(T)
 
-    idx = 1
     @inbounds for j in 1:K
         log_remainder = zero(T)  # log of proportion of unit vector remaining
         for i in 1:(j - 1)
             z = tanh(Y[i, j])
             W[i, j] = z * exp(log_remainder)
-            log_remainder += log1p(-z^2) / 2
+            log_remainder += log(2 / (exp(Y[i, j]) + exp(-Y[i, j])))
```

Comment:
AFAICT you can just use
```suggestion
            log_remainder -= logcosh(Y[i, j])
```
(assuming LogExpFunctions is loaded)